decidim-core/app/views/layouts/decidim/_decidim_javascript.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= javascript_pack_tag "decidim_core", defer: false %>
+<%= render partial: "layouts/decidim/js_configuration" %>
+
+<%= yield :js_content %>
+
+<%= snippets.display(:snippet_scripts) %>
```

Comment:
Same here, instead of naming this `:snippet_scripts`, I would rather name it `:foot`, so then we'd have `:head` and `:foot` which tell me where these appear in the DOM.

Also, this way you are duplicating the `:snippet_scripts` position here and in the `:head` as well:
https://github.com/decidim/decidim/blob/6bc5f814d14c577a7a2eba7b856e01ded6bba7c2/decidim-admin/app/views/layouts/decidim/admin/_header.html.erb#L7

So any snippets added to the `:snippet_scripts` "group" would be included twice, right?